packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -41,8 +39,7 @@ class Doctor {
       _validators.add(_iosWorkflow);
 
     List<DoctorValidator> ideValidators = <DoctorValidator>[];
-    ideValidators.addAll(AtomValidator.installed);
-    ideValidators.addAll(IntelliJValidator.installed);
+    ideValidators.addAll(IntelliJValidator.getInstalledValidators());
```

Comment:
using `get` prefix is so "Java" ;-) 
Perhaps rename `getInstalledValidators` --> `installValidators` ?